modules/moduledeps/deps.go

```diff
@@ -0,0 +1,9 @@
+package moduledeps
```

Comment:
Unfortunately, we are forced to put dependencies into a separate package, otherwise, we get a circular dependency issue: `modules/prebid/ortb2blocking` package depends on `modules` and `modules` depends on `modules/prebid/ortb2blocking` (we define all modules within the `modules/builder.go` file).